lib/Steps/Integrations/NextJs.ts

```diff
@@ -303,6 +319,32 @@ export class NextJs extends BaseIntegration {
     fs.writeFileSync(targetPath, filledTemplate);
   }
 
+  private _hasPackageInstalled(packageName: string): boolean {
+    const depsVersion = _.get(appPackage, ['dependencies', packageName]);
+    const devDepsVersion = _.get(appPackage, ['devDependencies', packageName]);
+    return !!depsVersion || !!devDepsVersion;
+  }
+
+  private _getPackageMangerChoice(): 'yarn' | 'npm' | null {
+    if (fs.existsSync(path.join(process.cwd(), 'yarn.lock'))) {
+      return 'yarn';
+    }
+    if (fs.existsSync(path.join(process.cwd(), 'package-lock.json'))) {
+      return 'npm';
+    }
+    return null;
+  }
```

Comment:
I recommend we add a case for [pnpm](https://pnpm.io/) here too since it seems very popular in Next.js community.